src/agent/github/build-agent.ts

```diff
@@ -22,6 +23,10 @@ class BuildAgent implements IBuildAgent {
         return toolCache.find(toolName, versionSpec, arch)
     }
 
+    public proxyConfiguration(url: string): IRequestOptions {
+        return undefined
```

Comment:
I wonder if you can fill is similar functionality for github as well

Reply:
I exported proxy settings to the github version, can you review it ?